### DISCUSSION TITLE
Dashboard search to return folders, not just dashboards

### DISCUSSION THREAD

**Discussion by samjewell** [grafana/grafana#40084] [Dashboards, Panels & Visualizations]
I have a folder named "Sam jewell", but it's hard to find on the /dashboards page. There are hundreds of folders there!
I often want to find my own folder using the "Search dashboards by name" input at the top of the "Manage" tab. Could that search be just a little closer to an "omni-search", and both return dashboards and folders 🙏 ?

As I write this I realise that I can use browser-search (cmd-F) on that page, to find my own folder. But somehow I expect that search functionality to work, and never thought to use cmd-F on this page until now - it's felt like a thorn in my side

From the FAQ: _Focus on the problem, not the solution:_
**The problem is** that I just want to find the dashboards that belong to me, but I can never remember what I named them (some have crappy names like "testing"). So I search for my name instead, as I know the dashboards I'm looking for live in the folder that has my name.

**Comment by samjewell** [comment]
As I was writing the above, I was reminded of another frustration in a similar part of the app. (Let me know if you want me to extract this comment into a separate discussion.)

When I click "Go to folder" I get to a nice page where I see just my own folder and my own dashboards. URL is `/dashboards/f/:uid/sam-jewell`. But if I take an action on that "filtered-list" page, such as deleting a dashboard, then as the action completes I'm thrown back to `/dashboards` - this is a context-switch I didn't want or ask for, and I have to repeat the process to get back to my own folder once more.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by samjewell** [comment]
This is now fixed!
